test/e2e/proxy_test.go

```diff
@@ -665,6 +665,7 @@ func TestSpaceLister(t *testing.T) {
 			identityID:            uuid.Must(uuid.NewV4()),
 		},
 	}
+	appStudioTierRolesWSoption := commonproxy.WithAvailableRoles([]string{"admin", "contributor", "maintainer"})
```

Comment:
I think that a proper name would be 
```suggestion
	appStudioTierRolesWSOption := commonproxy.WithAvailableRoles([]string{"admin", "contributor", "maintainer"})
```
;)

Reply:
yup , that's what I meant. 

done please see: https://github.com/codeready-toolchain/toolchain-e2e/pull/797/commits/1910a963765b2f448930f82b8cb2c4de84b84192